### PR TITLE
Use NuGet for the NUnit dependencies

### PR DESCRIPTION
**Pull request by jasonleenaylor**
* Enable package restore
* Use NuGet for each test project that uses NUnit
* Add NuGet targets and depend on the NuGet Restore
  for all test projects

### CODE REVIEW

**Comment by jasonleenaylor** [comment]
This will get the master builds of FLExBridge working again

**Comment by ermshiperete** [comment]
The msbuild integrated approach is no longer recommended by the NuGet developers. The other approach doesn't require a change in every .csproj file. 

Possibly you had a `.nuget` subdirectory when you opened the solution - this causes VS to use the msbuild integrated approach and modify the .csproj files.

File: .nuget/NuGet.targets [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ermshiperete) [inline .nuget/NuGet.targets]: In order to NOT use the msbuild integrated approach, this file has to reside somewhere else, e.g. as `build/NuGet.targets`

File: FLExBridge VS2010.sln [hunk 2]
```diff
@@ -30,6 +30,13 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TriboroughBridge-ChorusPlug
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RepositoryUtility", "src\RepositoryUtility\RepositoryUtility.csproj", "{FC668C72-7472-4774-8501-8E021ECF3BEA}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{E2661AE6-A7BA-41BA-A2CF-DECB376B2729}"
```
Comment (ermshiperete) [inline FLExBridge VS2010.sln]: Not necessary for Automatic Package Restore approach

File: src/FLEx-ChorusPluginTests/FLEx-ChorusPluginTests.csproj [hunk 3]
```diff
@@ -1,263 +1,275 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (ermshiperete) [inline src/FLEx-ChorusPluginTests/FLEx-ChorusPluginTests.csproj]: Changes not necessary for Automatic Package Restore approach

Comment (jasonleenaylor) [inline src/FLEx-ChorusPluginTests/FLEx-ChorusPluginTests.csproj]: I don't think this is true. For it to work from Visual Studio the package.config file needs to be in the solution somewhere.

Comment (jasonleenaylor) [inline src/FLEx-ChorusPluginTests/FLEx-ChorusPluginTests.csproj]: Also, the reference must change to the new location for the nunit dependency.

**Pull request merged**